buildtest/cli/__init__.py

```diff
@@ -1295,3 +1331,24 @@ def cdash_menu(subparsers):
     upload.add_argument(
         "-o", "--open", action="store_true", help="Open CDASH report in browser"
     )
+
+
+def add_hidden_parser(subparser, hidden_parser):
+    """This method adds a help message to certain parsers
+
+    Args:
+        subparsers (argparse._SubParsersAction): Subparser object
+        hidden_parser: dictionary
+    """
+    if "command" not in hidden_parser.keys():
+        for command, help_msg in hidden_parser.items():
+            subparser.add_parser(command, help=help_msg)
```

Comment:
Since you are building the parser here might as well capture the aliases too when building the parser so you don't need to call this method again.

Just have this method called once in such a way the dictionary is setup to take care of the logic.

For commands that don't have an alias then don't define it in the dictionary

Reply:
I just made these changes, please let me know if they're correct

Reply:
thanks